flipper-plugin/src/data/ClassInfo.tsx

```diff
@@ -1,13 +1,8 @@
+import {com} from "backintime-websocket-event";
+import PropertyInfo = com.kitakkun.backintime.core.websocket.event.model.PropertyInfo;
+
 export interface ClassInfo {
-  name: string;
+  classSignature: string;
   superClassName: string;
```

Comment:
we should also rename this

Reply:
Addressed in [Rename to signature](https://github.com/kitakkun/back-in-time-plugin/pull/143/commits/24a6ea3749a86d5a218c95c68d7fc960cb29bf32)